wormhole-connect/src/utils/transferValidation.ts

```diff
@@ -255,7 +256,8 @@ export const validate = async (dispatch: Dispatch<AnyAction>) => {
     transferInput.token &&
     transferInput.destToken &&
     transferInput.amount &&
-    Number.parseFloat(transferInput.amount) >= 0
+    Number.parseFloat(transferInput.amount) >= 0 &&
+    transferInput.availableRoutes !== undefined
```

Comment:
Ah I see. Hm, could you just check the length rather than make it optionally undefined?

Reply:
Wait, ignore that.  I looked at the issue a bit more closely.  This makes sense